ui/shared/components/panel/variants/VariantGene.jsx

```diff
@@ -98,28 +174,30 @@ const BaseLocusListLabels = React.memo((
   ) : (
     <div style={containerStyle}>
       {locusListGuids.map((locusListGuid) => {
-        const panelAppConfidence = locusListConfidence && locusListConfidence[locusListGuid]
-        let { description } = locusListsByGuid[locusListGuid] || {}
-        if (panelAppConfidence) {
-          description = (
-            <div>
-              {description}
-              <br />
-              <br />
-              <b>PanelApp gene confidence: &nbsp;</b>
-              {PANEL_APP_CONFIDENCE_DESCRIPTION[panelAppConfidence]}
-            </div>
-          )
+        const locusList = locusListsByGuid[locusListGuid]
+        const panelAppDetails = panelAppDetail[locusListGuid]
+        const { name: label, description: locusListDescription, paLocusList } = locusList
+        const { description, initials, customColor } = panelAppDetails && paLocusList ? getPaProps({
+          panelAppDetails,
+          panelAppPanel: locusListDescription || '',
```

Comment:
why not name this prop `locusListDescription ` to make it more readable

Reply:
The function that is receiving this property, PanelAppHoverOver() wants to know the PanelApp Panel (which is different to the PanelApp Name).

I thought the word description was becoming a bit too overloaded and wanted to be clear what is happening in that function, since it is returning a description.

I'm happy to use `locusListDescription`, `panelAppPanel` could be a bit confusing for a reader unfamiliar with the PanelApp datasource.